deb/openmediavault/var/www/openmediavault/js/omv/workspace/Workspace.js

```diff
@@ -45,14 +45,15 @@ Ext.define("OMV.workspace.Workspace", {
 		"OMV.toolbar.ApplyCfg",
 		"OMV.toolbar.HostnameItem",
 		"OMV.window.MessageBox",
-		"OMV.workspace.node.tree.Panel"
+		"OMV.workspace.node.tree.Panel",
+		'Ext.rtl.*'
 	],
 	uses: [
 		"OMV.workspace.node.panel.Category",
 		"OMV.workspace.node.panel.Overview",
 		"OMV.workspace.tab.Panel"
 	],
-
+	rtl: true,
```

Comment:
This will enable RTL for everyone. I don't think non-RTL users will be happy about that.

Reply:
I don't think so. It gives the needed support to render RTL. However, if the included js and css are not the RTL versions, extjs stays LTR.